app/src/main/java/explorer/city/com/cityexplorer/Model/AlternateName.kt

```diff
@@ -0,0 +1,5 @@
+package explorer.city.com.cityexplorer.Model
```

Comment:
Packages should be all lower-case (https://github.com/raywenderlich/kotlin-style-guide#packages). This is the same in Java as well.